### PR TITLE
Blair Englund

### PR DESCRIPTION
**Pull request by blairenglund**

### CODE REVIEW

File: lib/arrays.md [hunk 1]
```diff
@@ -5,44 +5,97 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+In Ruby an array is a collection of data. An array could contain multiple numbers or strings, each one having a position in the array.
```
Comment (halfghaninne) [inline lib/arrays.md:8]: Are there other data types that can be stored in an Array?

**Comment by halfghaninne** [comment]
👍

**Comment by halfghaninne** [comment]
Otherwise, 👍

File: lib/arrays.md (hunk 1 above)
Comment (blairenglund) [inline lib/arrays.md:8]: Yes! An array can contain another array. I suppose there's nothing stopping someone from making a matryoshka of nested arrays.

**Pull request closed without merging**